releasenotes/releasenotes.rst

```diff
@@ -24,6 +24,27 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+.. _vonk_releasenotes_421:
+
+Release 4.2.1 hotfix
+--------------------
+
+Database
+^^^^^^^^
+.. note::
+   This hotfix release is mainly for fixing a bug reported in version 4.2.0 of Firely Server SQL Server database, which affects the query performance.
+
+.. attention::
+   For SQL Server the foreign keys are made to be trusted. The related upgradescript (`data/20210720085032_EnableCheckConstraintForForeignKey.sql`) can take some time if you have many resources loaded in your database. Therefore some guidelines:
+
+   * We tested it on a database with about 15k Patient records, and 14 mln resources in total. Migrating that took about 20 seconds on a fairly powerful laptop.
+   * Absolutely make sure you create a backup of your database first.
+   * If you already expect the migration might failed or time out, you can run it manually upfront. Shut down Firely Server, so no other users are using the database, and then run the script from SQL Server Management Studio (or a similar tool).
+
+Fix
+^^^
+#. Fixed a bug about Not Trusted Foreign Key Constraints. This bug has an impact on the query performance since SQL Server optimizer won't trust these constants and to perform extra checks.
```

Comment:
```suggestion
#. Fixed a bug where some of the Foreign Keys in SQL Server had become untrusted. This bug has an impact on the query performance since the the SQL Server query optimizer will not consider FKs when they are not trusted. This has been fixed, all Foreign Keys have been validated and are trusted again.
```